### PR TITLE
CNV-36170: network policies empty state lists

### PR DESCRIPTION
**Pull request by upalatucci**
Adding lists of empty states

hide pagination component during loading and on empty state

Screenshot from 2024-03-27 16-30-22
Screenshot from 2024-03-27 16-30-19

### CODE REVIEW

File: src/views/networkpolicies/list/MultiNetworkPolicyList.tsx [hunk 1]
```diff
@@ -86,25 +88,28 @@ const MultiNetworkPolicyList: FC<MultiNetworkPolicyListProps> = ({ namespace })
...
-              onPaginationChange({ endIndex, page, perPage, startIndex })
-            }
-            page={pagination?.page}
-            perPage={pagination?.perPage}
-            perPageOptions={paginationDefaultValues}
-          />
+          {loaded && !isEmpty(data.length) && (
+            <Pagination
+              isLastFullPageShown
+              itemCount={data?.length}
```
Comment (hstastna) [inline src/views/networkpolicies/list/MultiNetworkPolicyList.tsx:94]: You don't need `?` here in `data?.length`. Or do you?

**Review by hstastna — COMMENTED** [review]
Looks good to me, it just does not make sense to display "create" buttons twice, in those list pages.. WDYT?

**Comment by upalatucci** [comment]
You're right. I'll fix that

**Comment by upalatucci** [comment]
@hstastna updated

**Review by hstastna — APPROVED** [review]

**Pull request merged**